lib/components/gene/header.js

```diff
@@ -46,7 +48,11 @@ class Header extends React.Component {
         </View>
       )
     } else {
-      return (<View style={styles.followButton}/>)
+      return (
+        <View style={styles.followButton}>
+          <InvertedButton text="Follow" textOpacity={0.5} />
+        </View>
```

Comment:
I’ve been wondering if we can have a ‘disabled because of still loading’ state for these follow buttons. OTOH, we should get that data in the GraphQL response, so in the future that would be moot.